hackathon_site/registration/models.py

```diff
@@ -56,12 +56,30 @@ class Application(models.Model):
         ("other", "Other"),
     ]
 
+    TSHIRT_SIZE_CHOICES = [
+        ("small", "S"),
```

Comment:
We can just use the same string in the database as in the form `("S", "S")`, etc